main.go

```diff
@@ -2,15 +2,15 @@ package main
 
 import (
 	"github.com/gin-gonic/gin"
-	"se-school-case/controller"
-	"se-school-case/initializer"
-	"se-school-case/service"
+	controller2 "se-school-case/pkg/controller"
```

Comment:
Names like `controller2` and `initializer2` should be avoided. 
Add something like this `pkgController` and `pkgInitializer` but I think 
`"se-school-case/pkg/controller"`
`"se-school-case/pkg/initializer"`
 will work since there are no duplicate names in the import